client/src/views/Groups.tsx

```diff
@@ -29,34 +34,113 @@ export const Groups = () => {
                     visibility: group.visibility,
                 }))
             )
-        })
+        } catch (e) {
+            console.log('error is', e);
+        }
+    }
+    useEffect(() => {
+        setGroupData()
     }, [])
 
+    const deletedGroups = useMemo(() => {
+        return groups.filter(group => group.deletedDateTime)
+    }, [groups])
+
+    const columnDefGroups: ColDef[] = [{ field: 'displayName', headerName: 'Display Name', width: 200, wrapText: true,
+    autoHeight: true}, { field: 'description', headerName: 'Description', flex: 10, wrapText: true, autoHeight: true }, 
```

Comment:
This is how it looking on my screen:
![image](https://github.com/adbahrani/MSGraphDashBoard/assets/41742806/0f68d48d-6497-4e5f-8963-d6497cc8f482)
Something is wrong here as description should be much bigger than the reset, somewhere width is taking over and it's not being adjusted. Also I don't think the text wrap is the best approach, rather have the ellipse .... with tool-tip when you hover ( I believe default behavior)